rclpy/src/rclpy/signal_handler.cpp

```diff
@@ -12,16 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <Python.h>
-#include <signal.h>
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
```

Comment:
style nit: shouldn't this be after all the includes?

Reply:
Rebased and moved namespace  line after includes in 0cbce013516dd1835d336822a70e8e7724476f9f